api/app.go

```diff
@@ -115,8 +115,17 @@ func (app *App) Configure() {
 }
 
 func (app *App) configureCache() {
-	cacheTTL := app.Config.GetDuration("cache.ttl")
-	app.cache = gocache.New(cacheTTL, gocache.DefaultExpiration)
+	// TTL
+	ttlKey := "cache.ttl"
+	app.Config.SetDefault(ttlKey, time.Minute)
+	ttl := app.Config.GetDuration(ttlKey)
+
+	// cleanup
```

Comment:
What `cleanup` means? I think this name is not intuitive for the person who is setting the configs. I only know what it is because I have read the cache lib documentation. Please choose another name or add a description.

Reply:
Sure! Done.